lib/yabeda/metric.rb

```diff
@@ -17,15 +17,15 @@ class Metric
 
     # Returns the value for the given label set
     def get(labels = {})
-      values[::Yabeda::Tags.build(labels)]
+      values[::Yabeda::Tags.build(labels, group)]
```

Comment:
I ran into some confusion at first that I thought `group` would be an instance of `Group` as opposed to a symbol.